site/en/docs/privacy-sandbox/attribution-reporting-changes-january-2022/index.md

```diff
@@ -2,7 +2,7 @@
 layout: 'layouts/doc-post.njk'
 title: 'Attribution Reporting proposal: updates in January 2022'
```

Comment:
```suggestion
title: 'Attribution Reporting proposal: updates'
```

Reply:
@maudnals since updates have continued since January, would this change make sense? We could add a line about "This was originally published with structural updates in January 2022, and we're continuing to update as the API changes"

Reply:
These changes are really fixes to the doc; the set of updates has not changed. This doc continues to be a snapshot of the January changes.
The Handbook is now the best place to read the API docs and already the info I've added in this PR, but one team in particular had this long-standing question about the January changes.